plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyOutputConsole.java

```diff
@@ -0,0 +1,153 @@
+package org.orbisgis.ui.editors.groovy;
+
+import org.eclipse.jface.action.MenuManager;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.custom.StyleRange;
+import org.eclipse.swt.custom.StyledText;
+import org.eclipse.swt.layout.FillLayout;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.ui.PlatformUI;
+import org.eclipse.ui.part.ViewPart;
+import org.jkiss.dbeaver.model.runtime.DBRProcessController;
+import org.jkiss.dbeaver.model.runtime.DBRProcessDescriptor;
+import org.jkiss.dbeaver.ui.ActionUtils;
+import org.jkiss.dbeaver.ui.UIUtils;
+import org.orbisgis.core.ui.ToolbarButton;
+
+/*
+ * This code is free software;
```

Comment:
You must used the same licence as here https://github.com/orbisgis/orbisrcp/blob/master/plugins/org.orbisgis.groovyeditor/src/org/orbisgis/ui/editors/groovy/GroovyActivator.java
and put it before package